dev/bots/test.dart

```diff
@@ -1004,6 +1004,25 @@ Future<void> _runFrameworkTests() async {
       // Web-specific tests depend on Chromium, so they run as part of the web_long_running_tests shard.
       '--exclude-tags=web',
     ]);
+    // Run java unit tests for integration_test
+    //
+    // Generate Gradle wrapper if it doesn't exists.
```

Comment:
To generate a wrapper reliability I think you should run flutter build apk --config-only

Reply:
I can change it to that, for context I got method (and comment) from here:
https://github.com/flutter/flutter/blame/5e6d88d766b91930f347f21417899a963eafc208/dev/tools/bin/generate_gradle_lockfiles.dart#L89

Reply:
It looks like you made this flag in the time since that way was written, I'll also put up a one liner to change that usage to `config-only` because it seems like the current way of building a flavor that doesn't exist is a hack that should be replaced by a flag being used in its intended way.

Reply:
https://github.com/flutter/flutter/pull/132967